pkg/tenant/topics.go

```diff
@@ -14,8 +14,46 @@ func (s *Server) ProjectTopicList(c *gin.Context) {
 	c.JSON(http.StatusNotImplemented, "not implemented yet")
 }
 
+// ProjectTopicCreate adds a new project topic to the database
+// and returns a 201 StatusCreated response.
+//
+// Route: /projects/:projectID/topics
 func (s *Server) ProjectTopicCreate(c *gin.Context) {
-	c.JSON(http.StatusNotImplemented, "not implemented yet")
+	var (
+		err   error
+		topic *api.Topic
+	)
+
+	// Bind the user request with JSON and return a 400 response
+	// if binding is not successful.
+	if err = c.BindJSON(&topic); err != nil {
+		log.Warn().Err(err).Msg("could not bind project topic create request")
+		c.JSON(http.StatusBadRequest, api.ErrorResponse("could not bind request"))
+		return
+	}
```

Comment:
Similar to `TenantProjectCreate`, do we need to be capturing the `projectID` param from the request URL and doing something with it to tie the topic to the project?

Reply:
A change was made in line with the comment above.